client/src/components/RatingsReviews/Breakdowns/ProductBreakdown.module.css

```diff
@@ -14,10 +14,16 @@ h2 {
   display: flex;
   flex-direction: column;
   /* padding: 0.5em 0.5em; */
-  font-weight: bold;
 
 }
 
+.characteristicText {
```

Comment:
<img width="286" alt="Screen Shot 2022-09-08 at 3 14 22 PM" src="https://user-images.githubusercontent.com/107006525/189235571-6170ef4e-3877-4b29-a885-ff76ea35e7e1.png">

For some reason I still see this